wcivf/apps/core/context_processors.py

```diff
@@ -27,6 +27,6 @@ def referer_postcode(request):
     referer_parts = referer_parts.strip("/").split("/")
     if len(referer_parts) >= 2 and referer_parts[-2] == "elections":
         possible_postcode = clean_postcode(referer_parts[-1])
-        if len(possible_postcode) < 8:
+        if len(possible_postcode) < 10:
```

Comment:
Wondering what the reasoning is behind this increase?

Reply:
Reverted. That was me playing around.